apps/marketplace/src/components/Sale.tsx

```diff
@@ -102,7 +103,7 @@ const Sale: FunctionComponent<SaleProps> = ({
 
     return {
       newmAmount: formatNewmAmount(newmAmount),
-      usdAmount: currency(usdAmount).format(),
```

Comment:
Is there a reason why currency was used here instead of  the formatUsdAmount?

Reply:
I think we just need to do a simple review of them and double check, will be part of the overall review and refactor of currencies across packages.